### DISCUSSION TITLE
new release tar.gz for docker

### DISCUSSION THREAD

**Discussion by celevra** [oposs/smtp-proxy#15] [Q&A] [answered]
Hi,

there was a new release 3 days before but no new smtp-proxy tar.gz to build the docker container.
Could you upload it or instruct me how to create the tar?

regards

**Comment by oetiker** [comment, marked as answer]
Hi @celevra 

I have updated the readme and added a build script for docker ... does this help ?

cheers
tobi

**Reply by celevra** [reply]
Hi tobi,

that does help, thank you very much

but i get following error in the docker build process

```
! Configure failed for Net-SSLeay-1.92. See /docker/smtp-proxy/thirdparty/work/1682609888.2882256/build.log for details.
! Installing the dependencies failed: Module 'Net::SSLeay' is not installed
! Bailing out the installation for IO-Socket-SSL-2.081.
! Installing the dependencies failed: Module 'IO::Socket::SSL' is not installed
! Bailing out the installation for /docker/smtp-proxy/.
```

regards

**Reply by celevra** [reply]
apt install libnet-ssleay-perl

and now it finishes, thank you again

**Reply by oetiker** [reply]
Actually what was missing on your system was not the perl module but most likely  `libssl-dev`
